subxt/src/rpc/rpc_client.rs

```diff
@@ -166,7 +166,8 @@ impl<Res> std::fmt::Debug for Subscription<Res> {
 }
 
 impl<Res> Subscription<Res> {
-    fn new(inner: RpcSubscription) -> Self {
+    /// Creates a new [`Subscription<Res>`].
```

Comment:
this type param in `doc links` is needless and could be simplified?!

```suggestion
    /// Creates a new [`Subscription`].
```